openshift/qontract-reconcile.yaml

```diff
@@ -1743,6 +1743,50 @@ objects:
           emptyDir: {}
         - name: fluentd-config
           emptyDir: {}
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    labels:
+      app: qontract-reconcile
+    name: qontract-reconcile-openshift-serviceaccount-tokens
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: qontract-reconcile
+    template:
+      metadata:
+        labels:
+          app: qontract-reconcile
+      spec:
+        securityContext:
+          runAsUser: ${{USER_ID}}
```

Comment:
Why do we need to template this? i.e. why don't we just let it run with a random UID?

Reply:
the openshift integrations are (sometimes) using ssh, which requires a valid user.
this is not a part of this PR though, it's the default templating.

i know it sucks.